### DISCUSSION TITLE
Goal detection?

### DISCUSSION THREAD

**Discussion by diegoparma** [roboflow/supervision#1011] [Q&A]
### Search before asking

- [X] I have searched the Supervision [issues](https://github.com/roboflow/supervision/issues) and found no similar feature requests.

### Question

Is possible to detect a football Goal?

**Comment by onuralpszr** [comment]
Hello, I am converting this to Q&A first.

**Comment by onuralpszr** [comment]
Hello, @diegoparma, yes you can detect Football Goal. If you can find a model that already has trained "football goal" you can use it. Or you can train yourself then you can use it. For example, If I do a simple search at Roboflow universe I can find a couple of results related to "Football Goal" (https://universe.roboflow.com/search?q=class%3Agoal) you can try one of those or you can find a different dataset elsewhere to train and use it. When it comes to visualization you can use "Supervision" to show it.   

I hope that helps.

**Comment by diegoparma** [comment]
Thanks you so much for the quick and good answer

**Reply by onuralpszr** [reply]
You are welcome. Please mark as answer

**Comment by diegoparma** [comment]
Thanks!
